clients/cloud/scala/src/main/scala/io/confluent/examples/clients/scala/Consumer.scala

```diff
@@ -0,0 +1,32 @@
+package io.confluent.examples.clients.scala
+
+import java.time.Duration
+import java.util.{Collections, Properties}
+
+import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import scala.collection.JavaConversions._
+
+  object Consumer extends App {
+    var conf = ConfigFactory.load("ccloud.properties")
+    val props = propsFromConfig(conf)
```

Comment:
should specify a group.id ("scala_example_group")

Reply:
done